### PR TITLE
Issue #24 - Fix so that server can be started in IDE.  Update README accordingly

### PR DESCRIPTION
**Pull request by chipmaurer**
Signed-off-by: Chip Maurer <chip.maurer@dell.com>

- Add jar statement in build.gradle to copy jar to correct plugin location
- Put sample properties files in etc
- Update README with correct instructions for IDE startup

### CODE REVIEW

File: etc/catalog/pravega.properties [hunk 1]
```diff
@@ -0,0 +1,3 @@
+connector.name=pravega
+pravega.controller=tcp://10.247.179.147:9090
```
Comment (adrdc) [inline etc/catalog/pravega.properties:2]: both these should be localhost..

Comment (chipmaurer) [reply etc/catalog/pravega.properties:2]: Yeah, I realized that.

**Review by adrdc — APPROVED** [review]

File: build.gradle [hunk 2]
```diff
@@ -54,6 +55,8 @@ dependencies {
     compile "com.facebook.presto:presto-spi:${prestoVersion}"
     compile "com.facebook.presto:presto-common:${prestoVersion}"
 
+    runtimeOnly group: 'io.netty', name: 'netty-all', version:"{nettyVersion}"
```
Comment (adrdc) [inline build.gradle:58]: i think maybe this needs to be compile ?  didn't seem to work for me w/ runtimeOnly for whatever reason

Comment (chipmaurer) [reply build.gradle]: Changed to 'compile'

**Review by adrdc — APPROVED** [review]

**Pull request merged**